# Farlik_2016_Example
Example data set using Farlik 2016 SC-WGBS data


# How to run

* Copy or link the Screw directory as a subdirectory of this one.
* Run run_farlik_pipeline.sh


* Note: we are having some trouble with Docker hashtags and quay.io, so this may or may not work. If you can get the latest build of the Docker image, with the latest Screw code contained within, it should run. 


Farlik 2016 Paper https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5145815/



Farlik2015 

Paper link: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542311/

Data link: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65196

They infer cell-state dynamics through analysis of collections of single-cell methylomes using a specific assay and bioinformatic method.
Studying the epigenomic cell-state dynamics in three in vitro models of cellular differentiation and pluripotency, they observed specific patterns of epigenome remodeling and cell-to-cell heterogeneity. They propose their methods can be used to study embryonic development, stem cell differentiation, and cancer, as well as generation of composite methylomes accounting for cell-to-cell heterogeneity in complex tissue samples.

Guo 2013 Paper http://genome.cshlp.org/content/23/12/2126

Guo 2013 Data https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47343

The authors describe a technique that enables single-cell and single-base resolution DNA methylation analysis using reduced representation bisulfite sequencing (scRRBS), which is highly sensitive and detects the methylation status of up to 1.5 million CpG sites within the genome of an individual mouse embryonic stem cell (mESC). They show that in a haploid sperm cell the technique detects the status of individual CpG site methylation in a digitized manner, either as unmethylated or fully methylated. They also show that tracing of the demethylation dynamics of maternal and paternal genomes after fertiliziation is possible within the individual pronuclei of mouse zygotes. Using their method they propose the ability to explore at single-base resolution the dynamic methylome landscapes of individual cells  in physiological and pathological processes, such as embryonic development, and even tumorigenesis.

Smallwood 2014 Paper https://www.nature.com/nmeth/journal/v11/n8/full/nmeth.3035.html

Smallwood 2014 Data https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1370522

The authors describe a method of single-cell bisulfite sequencing (scBS-seq) accurately measuring DNA methylation at up to 48.4% of CpG sites in embryonic stem cells grown in different media. They propose that scBS-seq is a versatile tool for exploring DNA methylation in rare cells and heterogeneous populations, integrating 12 individual mouse oocyte datasets and showing that it largely recapitulates the whole DNA methylome.
